### PR TITLE
Update continuous profiling.md

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Update continuous profiling

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/docs-cn/pull/7389

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by shichun-0415** [comment]
I've finished the self-check of this PR. PTAL. Thanks.

File: dashboard/continuous-profiling.md [hunk 1]
```diff
@@ -69,7 +77,7 @@ summary: TiDB Dashboard 持续性能分析功能 (Continuous Profiling)
 
 升级后，完成启动前检查。
 
-### 启动功能流程
+### 配置中控机和 TiDB Dashboard
 
 1. 在中控机上，通过 TiUP 添加 ng_port 配置项，并对 Prometheus 节点进行 reload 操作。
```
Comment (shichun-0415) [inline dashboard/continuous-profiling.md:82]:
```suggestion
1. 在中控机上，通过 TiUP 添加 ng_port 配置项。
```

File: dashboard/continuous-profiling.md [hunk 2]
```diff
@@ -69,9 +77,9 @@ summary: TiDB Dashboard 持续性能分析功能 (Continuous Profiling)
 
 升级后，完成启动前检查。
 
-### 启动功能流程
+### 配置中控机和 TiDB Dashboard
 
-1. 在中控机上，通过 TiUP 添加 ng_port 配置项，并对 Prometheus 节点进行 reload 操作。
+1. 在中控机上，通过 TiUP 添加 ng_port 配置项。
```
Comment (shichun-0415) [inline dashboard/continuous-profiling.md:82]:
```suggestion
1. 在中控机上，通过 TiUP 添加 ng_port 配置项。并对 Prometheus 节点进行 reload 操作。
```

File: dashboard/continuous-profiling.md [hunk 3]
```diff
@@ -69,9 +77,9 @@ summary: TiDB Dashboard 持续性能分析功能 (Continuous Profiling)
 
 升级后，完成启动前检查。
 
-### 启动功能流程
+### 配置中控机和 TiDB Dashboard
 
-1. 在中控机上，通过 TiUP 添加 ng_port 配置项，并对 Prometheus 节点进行 reload 操作。
+1. 在中控机上，通过 TiUP 添加 ng_port 配置项。并对 Prometheus 节点进行 reload 操作。
```
Comment (shichun-0415) [inline dashboard/continuous-profiling.md:82]:
```suggestion
1. 在中控机上，通过 TiUP 添加 ng_port 配置项，并对 Prometheus 节点进行 reload 操作。
```

**Review by lilyjazz — APPROVED** [review]

File: dashboard/continuous-profiling.md [hunk 4]
```diff
@@ -7,15 +7,23 @@ summary: TiDB Dashboard 持续性能分析功能 (Continuous Profiling)
...
 
 持续性能分析是一种从系统调用层面解读资源开销的方法。引入该方法后，TiDB 可提供数据库源码级性能观测，通过火焰图的形式帮助研发、运维人员定位性能问题的根因。
 
 该功能以低于 0.5% 的性能损耗，对数据库内部运行状态持续打快照（类似 CT 扫描），让原本“黑盒”的数据库变成“白盒”，具备更高的可观测性。该功能一键开启后自动运行，存储结果提供了保留时长的设定，过期的结果将会被回收，确保存储空间的有效利用。
 
+## 使用限制
+
+使用持续性能分析时，应留意如下使用限制：
+
+- 该功能在 TiDB 5.3.0 引入，需要升级到 5.3.0 及以上版本使用。
```
Comment (TomShawn) [inline dashboard/continuous-profiling.md:21]:
```suggestion
- 该功能在 TiDB v5.3.0 引入，需要升级到 5.3.0 及以上版本使用。
```

File: dashboard/continuous-profiling.md [hunk 5]
```diff
@@ -135,8 +145,9 @@ summary: TiDB Dashboard 持续性能分析功能 (Continuous Profiling)
 
 ## 停用持续性能分析
 
-1. 在 TiDB Dashboard 的**高级调试** > **实例性能分析** > **持续分析**页面，点击**设置**，进入设置弹窗。
-2. 关闭**启用功能**开关。
-3. 点击**保存** (Save) 按钮。
+1. 进入 TiDB Dashboard，选择**高级调试** (Advanced Debugging) > **实例性能分析** (Profile Instances) > **持续分析** (Continuous Profile)。
+2. 点击**设置** (Settings)，将**启用特性** (Enable Feature) 下方的开关关闭。
+3. 在弹出的**停用持续性能分析** (Disable Continuous Profiling Feature) 对话框中，选择**停用** (Disable)。
```
Comment (shichun-0415) [inline dashboard/continuous-profiling.md:150]:
```suggestion
3. 在弹出的**停用持续分析** (Disable Continuous Profiling Feature) 对话框中，选择**停用** (Disable)。
```

File: dashboard/dashboard-profiling.md [hunk 6]
```diff
@@ -11,7 +11,7 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-profiling/']
 
 可以通过以下两种方法访问实例性能分析页面：
 
-- 登录后，左侧导航条点击 **高级调试** > **实例性能分析** > **手动分析页面**：
+- 登录后，左侧导航条点击**高级调试** (Advanced Debugging) > **实例性能分析** (Profile Instances) > **手动分析页面** (Manually Profile)：
```
Comment (shichun-0415) [inline dashboard/dashboard-profiling.md:14]:
```suggestion
- 登录后，左侧导航条点击**高级调试** (Advanced Debugging) > **实例性能分析** (Profile Instances) > **手动分析** (Manually Profile)：
```

File: dashboard/continuous-profiling.md [hunk 7]
```diff
@@ -7,15 +7,23 @@ summary: TiDB Dashboard 持续性能分析功能 (Continuous Profiling)
 
 > **警告：**
 >
-> 持续性能分析功能作为实验特性，不建议在生产环境中使用。
-> 该功能在 TiDB 5.3.0 引入，需要升级到 5.3.0 及以上版本体验。
-> 该功能可在 x86 架构下支持 TiDB、TiKV、PD ，暂不支持 TiFlash；而在 ARM 框架下还未完全兼容，不可开启。
-> 该功能暂时只用于使用 TiUP 部署和升级的集群，不支持 TiDB Operator 或二进制包部署和升级的集群。
+> 持续性能分析目前为实验特性，不建议在生产环境中使用。
+>
```
Comment (qiancai) [inline dashboard/continuous-profiling.md:11]:
```suggestion
```

**Review by qiancai — APPROVED** [review]

**Pull request merged**